lua/astrocommunity/colorscheme/oxocarbon-nvim/oxocarbon-nvim.lua

```diff
@@ -0,0 +1,3 @@
+return {
+  "nyoom-engineering/oxocarbon.nvim",
+}
```

Comment:
```suggestion
return { "nyoom-engineering/oxocarbon.nvim" }
```

This is a bit of a nitpick, but just to keep things minimal and clean, I think this is better